personal/privacyidea/class_mfaAccount.inc

```diff
@@ -178,8 +178,8 @@ class mfaAccount extends plugin
         /* Call parent execute */
         plugin::execute();
 
-        if (!isset($this->parent)) {
-            $this->uid = $this->ui->uid;
+        if (!$this->is_account && $this->acl_is_writeable("")) {
+            $this->is_account = true;
```

Comment:
Ooops, this code bit had been touched in an earlier commit. Please rebase properly, so that ping-pong changes disappear.

Reply:
Squashed.